sympy/geometry/curve.py

```diff
@@ -31,13 +29,28 @@ class Curve(GeometrySet):
     function : list of functions
     limits : 3-tuple
         Function parameter and lower and upper bounds.
+    dimension: integer
+        Dimension of the curve (corresponding to the number of functions used in its definition).
+    length: number or sympy expression
+        Arc length of the curve over the limits.
+    tangent: Matrix
+        Tangent vector to the curve at an arbitrary point.
+    normal: Matrix
+        Normal vector to the curve at an arbitrary point.
+    binormal: Matrix
+        Binormal vector to the curve at an arbitrary point.
+    curvature: number or sympy expression
```

Comment:
sympy -> SymPy

(A SymPy Number is also an Expr, so we might simply say that curvature is a SymPy Expr.)
```
>>> isinstance(S(1), Expr)
True
```